### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by MacKenzieHnC** [microsoft/react-native-windows#11179] [General]
I just wanted to say thank you for you all having the best documentation on Native Module creation that I've seen anywhere. It's so thorough! Seriously, thank you.

**Comment by roxk** [comment]
Not about react native module but I'd also like to thank everyone involved, especially @asklar with his quick bug fixes and overall participation in the UWP community discord server.

**Reply by MacKenzieHnC** [reply]
Seriously. The amount of work that goes into this project is staggering.
